addons/tls13-compat-ff51/bootstrap.js

```diff
@@ -150,28 +183,31 @@ function install() {
     console.log("User has changed TLS max version. Skipping");
     return;
   }
-  
+
+  // Only ever want to run once.
+  gNewInstall = true;
+
```

Comment:
The reason this is necessary is because I had to move the deadman time to `startup()` (it doesn't ever fire if I do it in `install()` - I need to investigate why)

In the meantime what this does is:

1) `install()` only ever runs once, when the add-on is installed, and sets `gNewInstall`
2) `startup()` checks `gNewInstall` and only adds the timer if it is set

This way both `install()` and `startup()` only every run once. Otherwise, `startup()` will be called on every browser startup and we'll get a bunch of "timedout" telemetry pings.